src/bin/ipc_latency.rs

```diff
@@ -243,61 +310,70 @@ fn main() {
         .version("0.1.0")
         .author("Akshay Narayan <akshayn@mit.edu>")
         .about("Benchmark of IPC Latency")
-        .arg(
-            Arg::with_name("iterations")
-                .long("iterations")
-                .short("i")
-                .help("Specifies how many trials to run (default 100)")
-                .default_value("100"),
-        )
+        .arg(Arg::with_name("iterations")
+             .long("iterations")
+             .short("i")
+             .help("Specifies how many trials to run (default 100)")
+             .default_value("100"))
+        .arg(Arg::with_name("implementation")
+            .long("impl")
+            .help("Specifies the type of ipc being benchmarked")
+            .default_value("nl"))
```

Comment:
Instead of a string value here, it's probably best to use this: https://docs.rs/clap/2.32.0/clap/macro.arg_enum.html - that way the value of the string is automatically validated and we can match against it.

Also, should probably allow this arg to be given multiple times: https://docs.rs/clap/2.32.0/clap/struct.Arg.html#method.multiple

Reply:
Sounds good.